server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1475,22 +1475,20 @@ public EntitlementCertificate createUeberCertificate(@Context Principal principa
         Owner o = findOwner(ownerKey);
 
         if (o == null) {
-            throw new NotFoundException(i18n.tr(
-                "owner with key: {0} was not found.", ownerKey));
-        }
-
-        Consumer ueberConsumer = consumerCurator.findByName(o, Consumer.UEBER_CERT_CONSUMER);
-
-        // ueber cert has already been generated - re-generate it now
-        if (ueberConsumer != null) {
-            List<Entitlement> ueberEntitlement
-                = entitlementCurator.listByConsumer(ueberConsumer);
-            // Immediately revoke and regenerate ueber certificates:
-            poolManager.regenerateCertificatesOf(ueberEntitlement.get(0), true, false);
-            return entitlementCertCurator.listForConsumer(ueberConsumer).get(0);
+            throw new NotFoundException(i18n.tr("owner with key: {0} was not found.", ownerKey));
         }
 
         try {
+            Consumer ueberConsumer = consumerCurator.findByName(o, Consumer.UEBER_CERT_CONSUMER);
+            List<Entitlement> ueberEntitlements = entitlementCurator.listByConsumer(ueberConsumer);
```

Comment:
I've made the change, but for what it's worth, it's entirely unnecessary. The contract on listByConsumer is such that we know it always returns a list and has no input restrictions. The code flow doesn't change at all here, and we end up making it slightly more ugly/convoluted for no additional stability or security.